components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +213,17 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
+function getPlayerFlag(name)
```

Comment:
```suggestion
function matchFunctions:_getPlayerFlag(name)
```

Reply:
I don't understand this, is `p` like a namespace?